docs/UserGuide.md

```diff
@@ -632,6 +637,10 @@ Examples:
     * Deletes the 2nd appointment in the list of appointments.
     * Then, create a 5-hour appointment named "Contract Signing With Charlie" on 22nd Oct 2022 at 4:30 PM, associated with the first person in the contact list
 
+<div markdown="span" class="alert alert-warning">:rotating_light: **Important Note:**
+The `chain` operation is executed sequentially, it will fail until a given command executed failed and the error message of that error command will show, the previous successful command before will be saved. 
```

Comment:
This is not accurate
```suggestion
The `chain` operation is executed sequentially, it will execute the commands from left to right until a given command fails. Any error message shown is from the failing operation, and all preceding operations will be saved.
```

Reply:
thx changed